### DISCUSSION TITLE
Turkish Translations

### DISCUSSION THREAD

**Discussion by Zaryob** [Stellarium/stellarium#1879] [Translations]
Hi, I'm a engineer that uses stellarium. I just wanna try to use stellarium in my native lang, I see most of the parts not translated and some translations are missing. So I translated project from transifex (with AquilaNipalensis named account), and I'm presenting my work under this issue:

* I use Google Translate for basic sentences and find words equivalent
* I use Turkish Language Society (Türk Dil Kurumu) Dictionaries and TUBITAK Astronomy Guides to find exact translations for sky objects and standarts.
* I use Wikipedia pages about  Astronomy in Turkish

I recommend some thing in my translations, these are:
* We are not using english translations of lunar regions. Mostly sky enthusiasts that I communicating with, uses latin names of this regions to find details from wikis, and mostly they also use them in the data they collect. If there are any counter argument, I open for discussion. So I don't want to translate them from english versions, so I wrote them Latin names.
* Special names not changes in Turkish, we use it without translating mostly (If not ihave any equivalent included in our culture).
* For academical references part, I retained article and study names.
* No Turkish names were specified for some of the planetary figures. However, I made some suggestions using geography atlases. I also saw that Russian translations from other languages were made in this direction.
* In some description need translations I use paranthesis but transifex says there is an warning in here. I see this warnings not important in my case.

some changes i made on translated parts:
* Some nebula translations made as "nebula" not "bulutsu" ("bulutsu" is Turkish equivalent of nebula, mean as cloud like object)
* Pulsar means "atarca" in Turkish. In old translations there is a caos  because pulsar and atarca used at the same time. So I recommend changes. Just in exoplanetary test method named "Pulsar" given same as "Pulsar".
* Reflection nebula translated as "Yansıma bulutsusu" but it means in english reflecting nebula, It must be "yansı bulutsusu" as I recommended
* Acronycal/Cosmical/Helical rises/sets not have a exact translation but sayin them "Günaşırı" "Güniçeri" and "Günsarmal" is most logical way. This expression is all Sun related derived words. It means orderly (into daylight), (after daylight), (mixed in daylight and night).
* Name of the meteor showers are special names. Not translated, except month expression like "September"

**Comment by alex-w** [comment]
I converted the issue into discussion, because this is not a bug report, but a good plan to translate the planetarium and related stuff. I guess the similar plan may be applicable for other languages.

Thanks!

**Comment by alex-w** [comment]
@Zaryob could you write this message at Transifex also: https://www.transifex.com/stellarium/teams/80998/discussions/

**Comment by Zaryob** [comment]
Ok,  I send message to transifex.
